libyui-rake.gemspec

```diff
@@ -41,5 +41,5 @@ end
 
   # dependencies
   spec.add_dependency("rake", "~> 10.0")
-  spec.add_dependency("packaging_rake_tasks", "~> 1.0")
+  spec.add_dependency("packaging_rake_tasks", "~> 1.1.4")
```

Comment:
@mvidner do you think that rpm macro for ruby gems survive such double dependency?